frontend/src/components/login/Login.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useContext } from 'react';
 import { useHistory } from 'react-router-dom';
 import './Login.css';
-import UserContext from './UserContext';
+import UserContext from '../../helpers/UserContext';
```

Comment:
What is the idea behind /helpers/ ?

Reply:
Ok, we should probably come up with some kind of cross-application meaning for various words. In the back end, I started out using, "handlers" to just mean a general function that helps another function, but am trying to refactor things such that a handler only specifically serves http. So for helpers, we might want a convention which says they are helping in a specific way and document this. Is it specific to components? If we have another file such as something that does math but gets re-used, perhaps this might be a util/ for example.

Reply:
https://github.com/openpredictionmarkets/socialpredict/issues/67